File: Word Capitalization/word_capitalization.cpp
#include <iostream>
using namespace std;

string Word()
{
    string word;
    cin>>word;
    int first;
    first = word[0];
    if(first >= 97)
    word[0] = char(first-32);
    return word;

}
int main() {
	int remained_test_cases = 1 ;
// 	cin>>remained_test_cases;
	while(remained_test_cases > 0)
	{   cout<<Word();
	    remained_test_cases--;
	}
	return 0;
}

File: Word Capitalization/word_capitalization.py
def Word():
    word = list(input())
    if ord(word[0]) >= 97 :
        word[0] = chr(ord(word[0])-32)
    output = ""
    
    for w in word:
        output += w 
    return output    
    
    
remained_test_cases = 1 
# remained_test_cases = int(input())
while remained_test_cases > 0:
    print(Word())
    remained_test_cases -= 1 